### ISSUE TITLE
[FEATURE]: Vertical Layout for portrait setup

### ISSUE THREAD

**Issue by totorios99** [Bismuth-Forge/bismuth#397]
### Summary

It would be nice to have a row-based layout for vertical mounted monitors, a VerticalTile's Qtile like experience. https://docs.qtile.org/en/latest/manual/ref/layouts.html#verticaltile

### A clear and concise description of what you want to happen

The available height gets divided by the number of panes.

**Comment by krshrimali** [comment]
I agree, will be a nice feature to have. I'll try if I can implement this.

**Comment by fsimonis** [comment]
Wouldn't it be possible to automatically rotate the layouts for vertical screens? This would be a simple solution to this problem.
